### PR TITLE
Remove unnecessary Form Object attributes.

### PR DESCRIPTION
**Pull request by aliuk2012**
### What problem does this pull request solve?

We seem to be setting/passing in the current form and current page in most of the page form objects. They were not
being used in the form object and adding no value.

### Things to consider when reviewing

- Does it work when run on your machine?
- Is it clear what the code is doing?
- Do the commit messages explain why the changes were made?
- Are there all the unit tests needed?
- Has all relevant documentation been updated?

### CODE REVIEW

File: app/controllers/pages/date_settings_controller.rb [hunk 1]
```diff
@@ -44,7 +44,6 @@ def update
 private
 
   def date_settings_form_params
-    form = Form.find(params[:form_id])
```
Comment (aliuk2012) [inline app/controllers/pages/date_settings_controller.rb:47]: saves us making a request to forms-api to find the form

File: app/controllers/pages/name_settings_controller.rb [hunk 2]
```diff
@@ -46,8 +46,7 @@ def update
 private
 
   def name_settings_form_params
-    form = Form.find(params[:form_id])
```
Comment (aliuk2012) [inline app/controllers/pages/name_settings_controller.rb:49]: saves us making a request to forms-api to find the form

File: app/controllers/pages/text_settings_controller.rb [hunk 3]
```diff
@@ -44,7 +44,6 @@ def update
 private
 
   def text_settings_form_params
-    form = Form.find(params[:form_id])
```
Comment (aliuk2012) [inline app/controllers/pages/text_settings_controller.rb:47]: saves us making a request to forms-api to find the form

File: app/controllers/pages/type_of_answer_controller.rb [hunk 4]
```diff
@@ -80,8 +80,7 @@ def next_page_path(form, answer_type, action)
   end
 
   def answer_type_form_params
-    form = Form.find(params[:form_id])
```
Comment (aliuk2012) [inline app/controllers/pages/type_of_answer_controller.rb:83]: saves us making a request to forms-api to find the form

**Review by thomasiles — APPROVED** [review]
you are doing gods work 🙏

**Pull request merged**